src/thunderbird_accounts/authentication/utils.py

```diff
@@ -7,3 +47,13 @@ def is_email_in_allow_list(email: str):
         return True
 
     return email.endswith(tuple(allow_list.split(',')))
+
+
+def create_aia_url(action: KeycloakRequiredAction):
+    """Create a url for a user to start a keycloak flow.
+    These flows are defined as actions in KeycloakRequiredAction."""
+    redirect_uri = quote(get_absolute_url(reverse('login')))
+    return urljoin(
+        settings.KEYCLOAK_AIA_ENDPOINT,
+        f'?response_type=code&client_id=tb-accounts&kc_action={action.value}&redirect_uri={redirect_uri}',
```

Comment:
As you've mentioned on Matrix, the client_id here shouldn't be hardcoded and changing to the following works:

```suggestion
        f'?response_type=code&client_id={settings.OIDC_RP_CLIENT_ID}&kc_action={action.value}&redirect_uri={redirect_uri}',
```